src/app/AppContainer.js

```diff
@@ -48,22 +55,33 @@ type PropsType = {
 	switchMode: string => mixed,
 };
 
+type StateType = {
+	rootViewHeight: number,
+	safeAreaBottomInset: number,
+	isFullyBordered: boolean,
+}
```

Comment:
Feels like the AppContainer has picked up lots of implementation details (wrangling the height of a view for example, wrangling the post title, rendering the visual or html mode) while still having the responsibility to setup the editor and other things.

I wonder if that's too much for a component and it'd make sense to introduce a child component at this stage. Maybe re-introduce a `MainScreen` component like we had in the past. WDYT?

Reply:
Sure we could.
I think I took the easiest path for this PR because the layout part is intermixed with the RN bridge subscriptions, a bit of refactoring would help here and I thought it was outside the scope of the PR.

Reply:
Reason I think it's in scope here is because this PR (in its current form) removes the component separation we already had in place (between the AppContainer and BlockManager) and puts more stuff in the AppContainer. I think it would be best if we could keep AppContainer as "clean" as it was before. Trusted, it wasn't super clean anyway but, no reason to add more to it.

In essence, to introduce the component that will replace our older BlockManager.

Reply:
Ok, I'll it up ;)